pom.xml

```diff
@@ -313,4 +327,16 @@
     </dependency>
   </dependencies>
 
+  <profiles>
+    <profile>
+      <id>jdk17</id>
+      <activation>
+        <jdk>17</jdk>
+      </activation>
+      <properties>
+        <argLine>--add-opens=java.base/sun.security.util=ALL-UNNAMED</argLine>
+      </properties>
+    </profile>
+  </profiles>
```

Comment:
is this still needed? or was the need for it removed in https://github.com/jenkinsci/winstone/pull/203